recipe/emailverification/recipe.go

```diff
@@ -124,17 +127,15 @@ func recipeInit(config evmodels.TypeInput) supertokens.Recipe {
 			singletonInstance = &recipe
 
 			supertokens.AddPostInitCallback(func() error {
-				sessionRecipe, err := session.GetRecipeInstanceOrThrowError()
-				if err != nil {
-					return err
-				}
-
-				sessionRecipe.AddClaimFromOtherRecipe(evclaims.EmailVerificationClaim)
-
-				if config.Mode == "REQUIRED" {
-					sessionRecipe.AddClaimValidatorFromOtherRecipe(
-						evclaims.EmailVerificationClaimValidators.IsVerified(nil),
-					)
+				sessionRecipe := session.GetRecipeInstance()
```

Comment:
this is actually wrong. We want to return an error if the session recipe is not initialised.

Reply:
Done